tests/blackbox.py

```diff
@@ -131,21 +131,25 @@ def decorator_blackbox(func: SubroutineFnWrapper | ABIReturnSubroutine):
 
 # ---- API ---- #
 
+O = TypeVar("O")
```

Comment:
If we follow this approach, I recommend changing the type variable to another name such as `Y` our `Output` because the `O` looks to much like a zero to me, and this is confusing

Reply:
@tzaffi No problem - I made a change in https://github.com/algorand/pyteal/commit/685ae794536cb6d83ccb8fab1ff3d21b69bbdb89 and we may move away from the approach wholesale.  Resolving though feel welcomed to re-open if there's additional concerns.